x-pack/plugins/apm/common/elasticsearch_fieldnames.ts

```diff
@@ -106,3 +106,9 @@ export const TRANSACTION_TIME_TO_FIRST_BYTE =
   'transaction.marks.agent.timeToFirstByte';
 export const TRANSACTION_DOM_INTERACTIVE =
   'transaction.marks.agent.domInteractive';
+
+export const FCP_FIELD = 'transaction.marks.agent.firstContentfulPaint';
+export const LCP_FIELD = 'transaction.marks.agent.largestContentfulPaint';
+export const TBT_FIELD = 'transaction.experience.tbt';
+export const FID_FIELD = 'transaction.experience.fid';
+export const CLS_FIELD = 'transaction.experience.cls';
```

Comment:
Would it make sense to make a separate file with the csm fields to prepare for the imminent move out of APM?

Reply:
ok i will do this in a follow up PR.